cedar-policy-validator/src/human_schema/to_json_schema.rs

```diff
@@ -47,6 +47,20 @@ pub fn custom_schema_to_json_schema(
     Ok((SchemaFragment(fragment), warnings.into_iter()))
 }
 
+/// Convert a custom type AST into the JSON representation of the type.
+/// Conversion is done in an empty context.
+pub fn custom_type_to_json_type(ty: Node<Type>) -> Result<SchemaType, ToJsonSchemaErrors> {
```

Comment:
Why "custom" type? Isn't it just the internal Cedar type?

(Context: I was confused by the name at first because I was trying to figure out how it related to `commonType` definitions in the JSON / `type` declarations in the new schema syntax)

Reply:
Just copied the phrasing from another function in this file. We could perhaps do a consistency pass so that we call the new schema format one name everywhere.